pkg/mdformatter/linktransformer/link.go

```diff
@@ -197,9 +204,13 @@ type futureResult struct {
 
 // NewValidator returns mdformatter.LinkTransformer that crawls all links.
 // TODO(bwplotka): Add optimization and debug modes - this is the main source of latency and pain.
-func NewValidator(ctx context.Context, logger log.Logger, linksValidateConfig []byte, anchorDir string, reg *prometheus.Registry) (mdformatter.LinkTransformer, error) {
+func NewValidator(ctx context.Context, logger log.Logger, linksValidateConfig []byte, anchorDir string, storage *cache.Storage, reg *prometheus.Registry) (mdformatter.LinkTransformer, error) {
 	var err error
-	config := Config{}
+	defaultValidity, err := time.ParseDuration("120h")
```

Comment:
Why not directly create a `const` with default `time.Duration`? It would save us from extra parsing here :blush:

Reply:
Will do that!

Reply:
still not done?

Reply:
Yes, needs to be done!